fastscanner/adapters/realtime/redis_channel.py

```diff
@@ -0,0 +1,34 @@
+import logging
+from typing import Any, cast
+
+import pandas as pd
+import redis.asyncio as aioredis
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+class RedisChannel:
+    def __init__(
+        self,
+        host="localhost",
+        port=6379,
+        password=None,
+        db=0,
+        stream_key="realtime_stream",
+    ):
+        self.redis = aioredis.Redis(
+            host=host, port=port, password=password, db=db, decode_responses=True
+        )
+        self.stream_key = stream_key
+
+    async def push(self, data: list[dict[str, Any]]):
```

Comment:
```suggestion
    async def push(self, channel_id: str, data: list[dict[str, Any]]):
```
Let's pass the channel_id here and create a stream per symbol.